src/main/java/uk/nhs/cdss/config/MapperConfig.java

```diff
@@ -0,0 +1,28 @@
+package uk.nhs.cdss.config;
+
+import com.fasterxml.jackson.databind.DeserializationFeature;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.SerializationFeature;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+
+@Configuration
+public class MapperConfig {
+
+  /**
+   * Provides an {@see ObjectMapper} that is configured to serialise
+   * Java 8 java.time objects (Instant, LocalDate &c.) as ISO 8601 dates,
+   * behaviour which was not yet the default in our version of jackson.
+   * @return The mapper with the configured modifiers.
+   */
+  @Bean("enhanced")
```

Comment:
Think you only have the one mapper so doesn't need a qualifier?

Reply:
You're right, I knew there was a default `ObjectMapper` bean by default  but it turns out that was marked as `@ConditionalOnMissingBean` so I've removed the qualifier.